Language/Libraries_and_Scripts/Imports/semantics_A03_t01.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Let Ii be an import directive that refers to a URI via the string
+/// si. The semantics of Ii is specified as follows:
+/// Case ⟨Semantics of deferred imports⟩. If Ii is a deferred import with prefix
+/// p, a binding of p to a deferred prefix run-time namespace NSdeferred is
+/// present in the library namespace of the current library L. Let NSimport,i be
+/// the namespace imported from the library specified by Ii.
+/// NSdeferred then has the following bindings:
+/// ...
+/// • For every class, mixin, enum, and type alias declaration named id in
+///   NSimport,i, a corresponding getter named id with return type Type. Calling
+///   the getter results in a dynamic error.
+///
+/// @description Checks that for every class, mixin, enum, and type alias
+/// declaration in the imported library a corresponding getter with the same
+/// name and return type `Type` is in the current library. Call of this getter
+/// is a dynamic error
```

Comment:
I think it's a little bit confusing to say that the current library somehow contains those deferred imported entities (both the throwing ones before the `loadLibrary` future has completed, and the real ones that we have after that completion).

The specification says that the entities can be obtained by a look up in a _namespace_ which is associated with the deferred import prefix. This is exactly the same mechanism as the one which is used for an immediate import (they don't switch from "throwing" to "real" entities, but they do rely on a namespace to give access to the imported things).

So the deferred-imported entities are actually no more a part of the current library than all the immediately-imported entities, and we probably don't want to talk about the immediately-imported ones as being part of the current library.

So maybe:

```suggestion
/// @description Checks that for every class, mixin, enum, and type alias
/// declaration in the imported library a corresponding getter with the same
/// name and return type `Type` is present in the namespace denoted by the
/// deferred import prefix. An invocation of this getter is a dynamic error.
```